keyboards/gmmk/pro/rules.mk

```
@@ -0,0 +1,28 @@
+# MCU name
+MCU = STM32F303
+
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
```

We recommend the lite version of bootmagic, as it's less prone to "why does this key not work right" issues. 
```suggestion
BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
```